src/css/_global.css

```
@@ -273,6 +273,11 @@ html[data-theme="light"] .markdown img[src$=".svg"][alt*="chart"] {
   filter: invert(1) hue-rotate(200deg);
 }
 
+.markdown table code {
+  display: inline-block;
+  max-width: 100px;
```

```suggestion
  max-width: 25ch;
```
This makes things slightly wider while still allowing for overflow behaviour to be minimized.